src/cpp/r/include/r/RInternal.hpp

```diff
@@ -153,11 +153,7 @@
 #define R_bcDecode INTERNAL_R_FUNCTION
 #define R_ToplevelExec INTERNAL_R_FUNCTION
 #define R_RestoreHashCount INTERNAL_R_FUNCTION
-#define R_IsPackageEnv INTERNAL_R_FUNCTION
```

Comment:
You don't need to delete these functions if you want to use them from our code (see the comment at the top). Rather you can just do this in the translation unit where you want to use them:

```cpp
#define R_INTERNAL_FUNCTIONS 1
```

This should generally be done in a cpp file not an hpp file so as not to pollute other translation units with the definition.